api/apis/integration/service_binding_test.go

```diff
@@ -89,21 +89,26 @@ var _ = Describe("ServiceBinding Handler", func() {
 
 			serviceInstanceGUID = generateGUID()
 			_ = createServiceInstance(context.Background(), k8sClient, serviceInstanceGUID, spaceGUID, "service-instance-name", secretName)
-			req, err = http.NewRequestWithContext(ctx, "POST", serverURI("/v3/service_credential_bindings"), strings.NewReader(fmt.Sprintf(`{
-																																		 "relationships": {
-																																		 "app": {
-																																		 "data": {
-																																		 "guid": %q
-																																		 }
-																																		 },
-																																		 "service_instance": {
-																																		 "data": {
-																																		 "guid": %q
-																																		 }
-																																		 }
-																																		 },
-																																		 "type": "app"
-																																		 }`, appGUID, serviceInstanceGUID)))
+			req, err = http.NewRequestWithContext(
+				ctx,
+				"POST",
+				serverURI("/v3/service_credential_bindings"),
+				strings.NewReader(fmt.Sprintf(`{
+                    "relationships": {
+                        "app": {
+                            "data": {
+                                "guid": %q
+                            }
+                        },
+                        "service_instance": {
+                            "data": {
+                                "guid": %q
+                            }
+                        }
+                    },
+                    "type": "app"
+                }`, appGUID, serviceInstanceGUID)),
+			)
```

Comment:
The indentation looks a bit odd here. It appears this is a mix of tabs and spaces?

Reply:
Thanks, fixed with tabs